skyportal/models.py

```diff
@@ -112,24 +116,7 @@ class Source(Base):
 
     score = sa.Column(sa.Float, nullable=True)
 
-    ## pan-starrs
-    sgmag1 = sa.Column(sa.Float, nullable=True)
-    srmag1 = sa.Column(sa.Float, nullable=True)
-    simag1 = sa.Column(sa.Float, nullable=True)
-    objectidps1 = sa.Column(sa.BigInteger, nullable=True)
-    sgscore1 = sa.Column(sa.Float, nullable=True)
-    distpsnr1 = sa.Column(sa.Float, nullable=True)
-
-    origin = sa.Column(sa.String, nullable=True)
```

Comment:
`origin` is a good column to keep for provenance -- in fact, I'd suggest making this not nullable.

Reply:
@profjsb these lines were actually just moved up a little bit, not removed

Reply:
ok